src/rebar_prv_xref.erl

```diff
@@ -160,7 +160,8 @@ get_xref_ignorelist(Mod, XrefCheck) ->
     %% And create a flat {M,F,A} list
     lists:foldl(
       fun({F, A}, Acc) -> [{Mod,F,A} | Acc];
-         ({M, F, A}, Acc) -> [{M,F,A} | Acc]
+         ({M, F, A}, Acc) -> [{M,F,A} | Acc];
+         ({M}, Acc) -> [{M} | Acc]
```

Comment:
we should use `M ... when is_atom(N)` -- avoid tuples with a single element and just put the literal module in there instead

Reply:
Double checking here. Are you suggesting this for just the filter operation or for the xref ignores config spec as well?